packages/cardano-services/test/StakePoolSeach/StakePoolSearchHttpService.test.ts

```diff
@@ -508,11 +517,35 @@ describe('StakePoolSearchHttpService', () => {
           expect(response).toMatchSnapshot();
         });
 
-        it('if sort not provided defaults by name asc order', async () => {
+        it('if sort not provided defaults order by name and then by pool id asc', async () => {
           const response = await doServerRequest({});
+
+          const resultSortedCopy = [...response.pageResults].sort(sortByNameThenByPoolId);
+
+          expect(response.pageResults).toEqual(resultSortedCopy);
```

Comment:
Nice. I was thinking of just making two queries, one with the explicit sort and the other without, then compare, but this better communicates the expectation.